main.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from schemas import Author, AuthorCreate
+from database import SessionLocal
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(db: Session = Depends(get_db)):
+    return crud.get_all_authors(db=db)
+
+
+@app.get("/authors/{author_id}", response_model=Author)
```

Comment:
it's not  crucial, but it's generally recommended to finish endpoints with a '/'